frontend/src/views/components/Map/Toolbar/SearchArea/SearchArea.component.tsx

```diff
@@ -0,0 +1,195 @@
+      refetchOnMount: false,
+      refetchOnWindowFocus: false,
+      refetchOnReconnect: false,
+    },
+  );
+
+  const { predictions: places } = data || {};
+
+  const handleInputChange = useCallback(
+    (event: React.FormEvent<HTMLInputElement>) => {
+      setQuery(event.currentTarget.value);
+    },
+    [setQuery],
+  );
+
+  const handleChange = useCallback(
+    async (place: Place) => {
+      setSelectedPlace(place);
+
+      // Requesting geometry given coordinates
+      if (isCoordinates(place?.latLngString)) {
+        const { longitude, latitude } = place.latLngString.match(COORDINATES_REGEX).groups;
+        const { results } = await geocoderService.geocode({
+          location: { lat: Number(latitude), lng: Number(longitude) },
+        });
+        fitBounds(fromGeometryToPolygonBounds(results[0].geometry));
+        onAfterChange();
+      }
+
+      // Requesting geometry given a place id
+      else if (place?.place_id) {
+        const { results } = await geocoderService.geocode({ placeId: place.place_id });
+        fitBounds(fromGeometryToPolygonBounds(results[0].geometry));
+        onAfterChange();
+      }
+    },
+    [fitBounds, onAfterChange],
+  );
+
+  return (
+    <div className="m-search-map">
+      <Combobox value={selectedPlace} by="place_id" onChange={handleChange} nullable>
+        <Combobox.Input
+          onChange={handleInputChange}
+          displayValue={(place: Place) => place?.description || place?.latLngString}
+          className="search-combobox-input"
+          placeholder={t('Search by country, city, town, coordinates')}
+        />
+        <div className="search-combobox-options">
+          <Combobox.Options>
+            {places?.map((place) => (
+              <Combobox.Option key={place.place_id} value={place} as={React.Fragment}>
+                {({ active }) => (
+                  <li className={cx({ 'is-active': active })}>
+                    <span className="option-label">
+                      <HighlightedText text={place.description} highlight={query} />
+                    </span>
+                  </li>
+                )}
+              </Combobox.Option>
+            ))}
+            {places?.length === 0 && !isLoading && (
+              <div className="search-combobox-message">
+                <T _str="No results found" />
+              </div>
+            )}
+            {!places && isLoading && !isCoordinates(searchTerm) && !isNumber(searchTerm) && (
+              <div className="search-combobox-message">
+                <T _str="Loading..." />
+              </div>
+            )}
+            {!places && isCoordinates(searchTerm) && (
+              <Combobox.Option
+                value={{ place_id: null, description: null, latLngString: searchTerm }}
+                defaultChecked
+              >
+                <div className="search-combobox-input-coordinates">
+                  <span>Enter</span> to navigate coordinates
```

Comment:
This is not translated
```suggestion
                  <T _str={'{enter} to navigate coordinates'} enter={<span><T _str="Enter" _comment="Enter to navigate coordinates"/></span>} _comment="Enter to navigate coordinates" />
```